orbs/shared/jobs/release.yaml

```diff
@@ -24,6 +24,10 @@ parameters:
     description: The executor to use for the release
     type: executor
     default: "testbed-docker"
+  dryrun_timeout:
```

Comment:
The parameter could be set as more general `no_output_timeout`, so the non dry-run release step could utilize it as well.

Reply:
as long as we're confident they take roughly the same time i don't see an issue with this